deepchem/models/losses.py

```diff
@@ -63,12 +63,19 @@ def _create_pytorch_loss(self):
 
     def loss(output, labels):
       output, labels = _make_pytorch_shapes_consistent(output, labels)
-      return torch.nn.functional.smooth_l1_loss(
-          output, labels, reduction='none')
+      return torch.nn.functional.smooth_l1_loss(output,
+                                                labels,
+                                                reduction='none')
 
     return loss
 
 
+def l2_loss(output, labels):
```

Comment:
I think moving the definition out of the function is fine, if that avoids problems with lightning.  Let's rename it to `_pytorch_l2_loss()` to clarify that it's private and specific to PyTorch.